src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -863,7 +863,7 @@ private function loadDeleteForm(): void
         $this->template->assign('deleteForm', $deleteForm->createView());
     }
 
-    private function getHiddenJsonField(string $name, string $json): SpoonFormHidden
+    private function getHiddenJsonField(string $name, ?string $json = null): SpoonFormHidden
```

Comment:
I feel the `= null` provides some extra clarity even though it's not 100% necessary.

The add message also has this method but the json is never null, since you add the blocks yourself in which case the json is filled